src/functionalTest/resources/features/F-004/F-004.feature

```diff
@@ -0,0 +1,118 @@
+    When  a request is prepared with appropriate values
+    And   the request [for a malformed caseTypeId]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-042
+  Scenario: must get an error response for a without caseTypeId parameter in request
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [without caseTypeId parameter]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-043
+  Scenario: must get an error response for a malformed jurisdictionId
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a malformed jurisdictionId]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-044
+  Scenario: must get an error response for without jurisdictionId parameter in request
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [without jurisdictionId parameter]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-045
+  Scenario: must get an error response for a non-existing classification
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a non-existing classification]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-046
+  Scenario: must get an error response for a malformed classification
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a malformed classification]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-047
```

Comment:
roles parameter might be removed from our API spec. so this scenario will become obsolete later.

Reply:
at the moment we will keep this once its conformed will removed